subclass/Arbiter; Oath of Sanctuary .json

```
@@ -0,0 +1,264 @@
+				"Aura of Peace|Paladin||Sanctuary|Sanctuary Random Finds|7",
+				"Impenetrable Bastion|Paladin||Sanctuary|Sanctuary Random Finds|15",
+				"Seraph of Sanctuary|Paladin||Sanctuary|Sanctuary Random Finds|20"
+			]
+		}
+	],
+	"subclassFeature": [
+		{
+			"name": "Oath of Sanctuary",
+			"source": "Sanctuary Random Finds",
+			"page": 1,
+			"className": "Paladin",
+			"classSource": "PHB",
+			"subclassShortName": "Sanctuary",
+			"subclassSource": "Sanctuary Random Finds",
+			"level": 3,
+			"entries": [
+				"Paladins who swear the Oath of Sanctuary dedicate themselves to the protection of the weak, the preservation of life, and the maintenance of peace. They believe that a true warrior is not one who seeks battle, but one who stands firm in defense of those who cannot protect themselves. These paladins use their divine gifts to create sanctuaries of safety and peace wherever they tread, shielding others from harm and bringing calm to even the most dangerous situations.",
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Tenets of Sanctuary|Paladin||Sanctuary|Sanctuary Random Finds|3"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Oath Spells|Paladin||Sanctuary|Sanctuary Random Finds|3"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Channel Divinity: All Eyes on Me|Paladin||Sanctuary|Sanctuary Random Finds|3"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Channel Divinity: Desperate Protection|Paladin||Sanctuary|Sanctuary Random Finds|3"
+				}
+			]
+		},
+		{
+			"name": "Tenets of Sanctuary",
+			"source": "Sanctuary Random Finds",
+			"page": 1,
+			"className": "Paladin",
+			"classSource": "PHB",
+			"subclassShortName": "Sanctuary",
+			"subclassSource": "Sanctuary Random Finds",
+			"level": 3,
+			"header": 1,
+			"entries": [
+				"The tenets of the Oath of Sanctuary emphasize the protection of life and the pursuit of peace above all else. Paladins who swear this oath abide by these core principles:",
+				{
+					"type": "entries",
+					"entries": [
+						{
+							"name": "Protection",
+							"entries": [
+								"Your first and foremost responsibility is to protect everyone in your immediate care. You must remain at the side of those entrusted to your care rather than seek revenge or justice."
+							]
+						},
+						{
+							"name": "At All Costs",
+							"entries": [
+								"You must do everything possible to protect those in your care, even if it means sacrificing yourself or others outside your care. Putting those under your care comes above you at all costs."
+							]
+						},
+						{
+							"name": "Loyalty",
+							"entries": [
+								"Once you have committed yourself to protecting an individual, you must protect them until the terms of your contract or other agreement are done. You cannot willingly end it."
+							]
+						}
```

The subentries in Tenets of Sanctuary are missing types:
```suggestion
						{
							"name": "Protection",
							"type": "entries",
							"entries": [
								"Your first and foremost responsibility is to protect everyone in your immediate care. You must remain at the side of those entrusted to your care rather than seek revenge or justice."
							]
						},
						{
							"name": "At All Costs",
							"type": "entries",
							"entries": [
								"You must do everything possible to protect those in your care, even if it means sacrificing yourself or others outside your care. Putting those under your care comes above you at all costs."
							]
						},
						{
							"name": "Loyalty",
							"type": "entries",
							"entries": [
								"Once you have committed yourself to protecting an individual, you must protect them until the terms of your contract or other agreement are done. You cannot willingly end it."
							]
						}
```

---

Oh whoops